pkg/ldd/ldso_freebsd.go

```diff
@@ -0,0 +1,48 @@
+package ldd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const ldso = "/libexec/ld-elf*.so.*"
+
+func LdSo() (string, error) {
```

Comment:
Could this function be in common since the only thing that changes is the lsdo regexp?